LiteEditor/cl_editor.cpp

```diff
@@ -1501,6 +1504,7 @@ void clEditor::OnSciUpdateUI(wxStyledTextEvent& event)
 
         if ((m_statusBarFields & kShowSelectedChars) && selectionSize) {
             message << (!message.empty() ? ", " : "") << "Sel " << selectionSize;
+            message << (!message.empty() ? ", " : "") << "SelLn " << selectionLn;
```

Comment:
`message` cannot be `empty` with above line.

Reply:
True :-)